browser_use/browser/context.py

```diff
@@ -264,6 +264,10 @@ async def _create_context(self, browser: PlaywrightBrowser):
 			# Connect to existing Chrome instance instead of creating new one
 			context = browser.contexts[0]
 		else:
+			# camofoux
+			if True:
+				return await browser.new_context()
```

Comment:
```suggestion
```

Returning a browser without the config params below would break a bunch of behavioral stuff, they're not just there for stealth.